src/core_plugins/metrics/public/components/code.js

```diff
@@ -0,0 +1,10 @@
+import { EuiCode, EuiTextColor } from '@elastic/eui';
+import React from 'react';
+export function Code({ children }) {
+  return (
+    <EuiTextColor color="accent">
+      <EuiCode>{children}</EuiCode>
+    </EuiTextColor>
```

Comment:
It might be worthwhile to stick to the code styling provided by EUI? Maybe they are open to changing the code color in general if you feel it should be different.

Reply:
@elastic/kibana-design ^

Reply:
This went back and forth. It actually started with that accent color, but feedback was that it was too heavy so we went with the neutral github style.

I'd recommend sticking with the defaults and not overwriting it. Then if we do change it, the change would carry down.

cc @elastic/eui-design who will laugh at this one!